html/semantics/document-metadata/interactions-of-styling-and-scripting/conditionally-block-rendering-on-link-media-attr.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+
+<link rel=stylesheet href=stylesheet.py>
+<link rel=stylesheet media="screen and (max-width:10px)" href=stylesheet.py?stylesNotMatchingEnvironment&delay=2>
+<h1>Dominic Farolino</h1>
+<script>
+  test(() => {
+    const h1 = document.querySelector('h1');
+    const computedColor = getComputedStyle(h1).color;
+    const expectedColor = "rgb(128, 0, 128)";
+
+    assert_equals(computedColor, expectedColor);
+    assert_equals(document.styleSheets.length, 1);
+  }, "Only the style sheet loaded via a link element whose media attribute matches the environment should block following script execution");
+
+  const secondStylesheetTest = async_test("Two style sheets should be registered as style sheets for the document after 3 seconds");
+  step_timeout(() => {
```

Comment:
Shouldn't this be secondStylesheetTest.step_timeout?

Reply:
Hm that would make sense, however it also seems to work as it is, is that just a lucky accident?

Reply:
Yeah, in that case they won't be associated with this test, but with an unnamed global test. It's a little confusing, but helps reducing boilerplate for files containing a single test I believe.